### ISSUE TITLE
Ability to duplicate an event

### ISSUE THREAD

**Issue by nmorduch** [littleweaver/django-brambling#299]

**Comment by melinath** [comment]
This is really closely tied to #301. It is slightly lower-priority.

**Comment by melinath** [comment]
Once this is done, we should probably (in a separate ticket) remove the "event defaults" section & support. I think that path was a mistake... more confusing than helpful.

**Comment by melinath** [comment]
I had been thinking of adding a "duplicate" button to the event list item, but I think it might actually be better to do something more like the saved attendee workflow – on the "Create an event" page, if they already have events, offer two choices: either copy an event or start from scratch. Copying an event presents them with a couple fields: choose event (Chosen field), new event name, new event start / end times. The rest they can edit later. Copy all the config over. If there are custom dates, use a diff between them and the old start date to determine the new date.
